anypoint-mq/4.x/antora.yml

```diff
@@ -0,0 +1,17 @@
+name: anypoint-mq-connector
+version: 4.x
+display_version: 4.x (Mule 4)
+title: Anypoint MQ Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Provides publish-subscribe messaging to Mule apps using the Anypoint MQ cloud service.
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: anypoint-mq-connector
+    page-runtime-version: 4.1.1
```

Comment:
where is 4.1.1 picked up from? Is this same as the connector release version ? (in that case it should be 4.0.0.)

Reply:
@kparadkar-ms - that's the Mule version compatibility. I didn't change it from 3.x.